backends/gstreamer/render-unix/lib.rs

```diff
@@ -280,7 +269,7 @@ impl Render for RenderUnix {
         *self.gl_upload.lock().unwrap() = loop {
             match iter.next() {
                 Ok(Some(element)) => {
-                    if "glupload" == element.get_factory().unwrap().get_name() {
+                    if "glupload" == element.factory().unwrap().name() {
```

Comment:
```suggestion
                    if Some("glupload") == element.factory().and_then(|f| f.name()) {
```

or so. Not every element has a factory. Maybe also in other places

Reply:
It seems this is the last place. 😅